notify/notify.go

```diff
@@ -902,6 +909,7 @@ func (tms TimeMuteStage) Exec(ctx context.Context, l log.Logger, alerts ...*type
 
 	// If the current time is inside a mute time, all alerts are removed from the pipeline.
 	if muted {
+		tms.metrics.numAlertsSuppressedTotal.Add(float64(len(alerts)))
 		level.Debug(l).Log("msg", "Notifications not sent, route is within mute time")
```

Comment:
```suggestion
		level.Debug(l).Log("msg", "Notifications not sent, route is within mute time", "alerts", len(alerts))
```

Reply:
Now that we're here let's try and log the number of alerts we suppressed for traceability.

Reply:
added in e15292f